rust/opendp/src/core.rs

```diff
@@ -23,7 +23,7 @@
 use std::ops::{Div, Mul};
 use std::rc::Rc;
 
-use crate::error::Fallible;
+use crate::error::*;
```

Comment:
Why did this import appear in many files? Is it to make life easier when using the error macros?

Reply:
I switched things when integrating the ExplainUnwrap trait. Seems like we typically make use of pretty much everything in that module. Agreed that glob imports are scary though.

Reply:
Ah makes sense thanks.